CRM/Upgrade/Incremental/sql/4.7.18.mysql.tpl

```diff
@@ -68,3 +68,12 @@ UPDATE `civicrm_state_province` SET `name`='Uttarakhand', `abbreviation`='UT' WH
 UPDATE `civicrm_state_province` SET `name`='Yunlin County' WHERE `name` = 'Yunlin Conuty';
 UPDATE `civicrm_country` SET `name`='Palestine, State of' WHERE `name` = 'Palestinian Territory';
 UPDATE `civicrm_country` SET `name`='Virgin Islands, British' WHERE `name` = 'Virgin Islands,British';
+
+-- CRM-19464 renumber address options to add in 'Supplemental Address 3'
+SELECT @option_group_id_adOpt := max(id) from civicrm_option_group where name = 'address_options';
+UPDATE civicrm_option_value SET value = value + 1, weight = weight + 1 WHERE option_group_id = @option_group_id_adOpt AND value > 3;
+INSERT INTO
+ `civicrm_option_value` (`option_group_id`, {localize field='label'}label{/localize}, `value`, `name`, `grouping`, `filter`, `is_default`, `weight`, `description`, `is_optgroup`, `is_reserved`, `is_active`, `component_id`, `visibility_id`, `icon`)
+VALUES
+  (@option_group_id_adOpt, {localize}'{ts escape="sql"}Supplemental Address 3{/ts}'{/localize}, 4, 'supplemental_address_3', NULL, 0, NULL, 4, NULL, 0, 0, 1, NULL, NULL, NULL);
```

Comment:
There is a merge conflict here (due to some other upgrade code that's been merged into this file.
But in general I agree with @eileenmcnaughton we should use the `ensureOptionValueExists` function instead of raw sql.
Also I'm not sure that it's necessary to renumber the option values. Is it a problem to have them numbered as-is?